docs/docs/developers/sdk/Tracks.md

```diff
@@ -0,0 +1,653 @@
+| :------ | :--------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------- | :--------- |
+| `genre` | `Genre` (can be imported from `@audius/sdk`)                     | If provided, the top 100 trending tracks of the genre will be returned                                       | _Optional_ |
+| `time`  | `GetTrendingTracksTimeEnum` (can be imported from `@audius/sdk`) | A time range for which to return the trending tracks. Default value is **GetTrendingTracksTimeEnum.AllTime** | _Optional_ |
+
+#### Returns
+
+The return type is the same as [`getBulkTracks`](#getbulktracks)
+
+### getUndergroundTrendingTracks
+
+#### getUndergroundTrendingTracks(`params`)
+
+Get the top 100 trending underground tracks on Audius.
+
+Example:
+
+```typescript
+const { data: tracks } = await audiusSdk.tracks.getUndergroundTrendingTracks();
+console.log(tracks);
+```
+
+#### Params
+
+Optionally create an object with the following fields and pass it as the first argument.
+
+| Name     | Type     | Description                                                                  | Required?  |
+| :------- | :------- | :--------------------------------------------------------------------------- | :--------- |
+| `limit`  | `number` | If provided, will return only the given number of tracks. Default is **100** | _Optional_ |
+| `offset` | `number` | An offset to apply to the list of results. Default value is **0**            | _Optional_ |
+
+#### Returns
+
+The return type is the same as [`getBulkTracks`](#getbulktracks)
+
+---
+
+### searchTracks
+
+#### searchTracks(`params`)
+
+Search for tracks.
+
+Example:
+
+```typescript
+const { data: tracks } = await audiusSdk.tracks.searchTracks({
+  query: "skrillex",
+});
+console.log(tracks);
+```
+
+#### Params
+
+Create an object with the following fields and pass it as the first argument, as shown in the example above.
+
+| Name    | Type     | Description             | Required?    |
+| :------ | :------- | :---------------------- | :----------- |
+| `query` | `string` | The query to search for | **Required** |
+
+#### Returns
+
+The return type is the same as [`getBulkTracks`](#getbulktracks)
+
+---
+
+### streamTrack
+
+#### streamTrack(`params`)
+
+Get the url of the track's streamable mp3 file.
+
+Example:
+
+```typescript
+const url = await audiusSdk.tracks.streamTrack({
+  trackId: "PjdWN",
+});
+const audio = new Audio(url);
+audio.play();
+```
+
+#### Params
+
+Create an object with the following fields and pass it as the first argument, as shown in the example above.
+
+| Name      | Type     | Description         | Required?    |
+| :-------- | :------- | :------------------ | :----------- |
+| `trackId` | `string` | The ID of the track | **Required** |
+
+TODO: should we document premium content signature?
```

Comment:
We can do it in a collapsible section maybe

Reply:
I like that!